src/days-2-4-adv/adv.py

```diff
@@ -49,3 +50,28 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+while True:
+    print (p.__str__())
+    cmd = input("What do you want to do? You can: move n, move e, move s, move w, press q to quit: ")
+    if cmd == "q":
+        break
+    elif cmd == "n":
+        p.room = room[p.room].n_to
```

Comment:
Try something like this. 
`elif cmd == "n" and hasattr(p.room, 'n_to'):
        p.room = p.room.n_to`
https://docs.python.org/3/library/functions.html#hasattr